frontend/front/package.json

```diff
@@ -59,6 +59,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
What is this being used for?

Reply:
The eslent? message said to add that to pass the check. Said that it was most likely not to be fixed.